test/test_tableMapper.js

```diff
@@ -0,0 +1,55 @@
+var assert = require('assert');
+var mapper = require('../mappers/tableMapper');
+var mongoose = require('mongoose');
+const APP_DB = 'mongodb://127.0.0.1/eventplanner_db';
+
+describe('tableMapper testing suite', function() {
+	//After all tests reconnect to APP_DB
+	before(function(){
+		mongoose.connect(APP_DB);
+	});
+
+	it('should create a new table', function(done) {
+		mapper.createTable(12,[],5,['a','b','c','d','e'],1,function(err,res){
```

Comment:
We really shouldn't be creating testing data with production data.  In production its highly likely we'll have an event with event_id, if they have 12 tables then the 12th table is corrupted and deleted